about.html

```diff
@@ -0,0 +1,156 @@
+        Consectetur, neque necessitatibus dolores, aut incidunt veritatis ducimus unde doloribus ex quod dolorum eveniet
+        laborum vero ipsam. Vero quos velit autem labore?
+      </p>
+    </div>
+    <div class="top contact-us">
+      <p>Please contact us per email for any further questions about University Campus Hour 2015!</p>
+      <a href="mailto:universitycampushour@university.edu"><strong>universitycampushour@university.edu <i
+            class="fa fa-chevron-right" aria-hidden="true"></i></strong></a>
+    </div>
+  </section>
+
+  <section class="logo-section flex">
+    <h2>University Campus Hour 2022 Logo</h2>
+    <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Et voluptatibus quia quas fugiat mollitia, sint quisquam
+      harum consequatur expedita esse architecto sed blanditiis, nulla repudiandae inventore. Officiis, porro odio magni
+      cum, perferendis asperiores labore nostrum quisquam molestias cupiditate a corporis.</p>
+    <img src="images/logo.png" alt="University Campus Hour Logo">
+    <hr class="full-hr">
+  </section>
+
+  <section class="past-programs flex">
+    <h2>See the past University Campus Hours</h2>
+    <p>Take a look at the last two University Campus Hours which took place in New York and Las Vegas</p>
+    <div class="past-programs-list flex">
+      <div class="program1 flex pointer">
+        <h2>2021</h2>
+        <h5>University Campus Hour in New York</h5>
+      </div>
+      <div class="program2 flex pointer">
+        <h2>2020</h2>
+        <h5>University Campus Hour in Las Vegas</h5>
+      </div>
+    </div>
+  </section>
+
+  <section class="partners flex top" id="partners">
+    <h2 class="main-program-header top">Partners</h2>
+    <hr class="horizontal-line">
+    <div class="partner-items flex">
+      <div class="partner-icon">
+        <i class="fa fa-windows" aria-hidden="true"></i>
+      </div>
+      <div class="partner-icon">
+        <i class="fa fa-google" aria-hidden="true"></i>
+      </div>
+      <div class="partner-icon">
+        <i class="fa fa-apple" aria-hidden="true"></i>
+      </div>
+      <div class="partner-icon">
+        <i class="fa fa-amazon" aria-hidden="true"></i>
+      </div>
+      <div class="partner-icon">
+        <i class="fa fa-behance" aria-hidden="true"></i>
+      </div>
+    </div>
+  </section>
+
+  <footer class="footer flex top" id="contact">
+    <h3 class="featured-clubs-header">Contact</h3>
+    <hr class="horizontal-line">
+    <div class="flex contact-details">
+      <img src="images/small_logo.png" alt="university logo" class="ytu-logo">
+      <div class="adress">
+        <p class="phone-number"><b>Phone:</b> +123456789</p>
+        <p class="adress-details">
+          <b>Address:</b> Lorem, ipsum. <br>
+          Lorem ipsum dolor sit.<br>
+          34220 Lorem, ipsum dolor.
+        </p>
+      </div>
+    </div>
+    <p class="rights"><i class="fa fa-copyright"></i>2021. All rights reserved</p>
```

Comment:
- [ ]  As stated the project requirement The pages should look almost identical to the original design. So Kindly aligned the   "logo"  the `left` and "text" aligned to the `right` and adjust background color of the footer Just like the [design guidelines](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) 

The Design Guidelines Look likes this 

![Capture](https://user-images.githubusercontent.com/93598189/203609151-e05a101f-18e7-4d88-993d-f2abcfe076ce.PNG)


Your Project Design Look likes this 

![Capture](https://user-images.githubusercontent.com/93598189/203607832-90b9e9a1-ad8b-43ae-bb39-d9d1054e3e13.PNG)